### DISCUSSION TITLE
privacy resources

### DISCUSSION THREAD

**Discussion by atomGit** [undergroundwires/privacy.sexy#245] [General]
a few resources that people may not know about...

* [Lissy93/awesome-privacy: 🦄 A curated list of privacy & security-focused software and services](https://github.com/Lissy93/awesome-privacy)
* [Lissy93/personal-security-checklist: 🔒 A compiled checklist of 300+ tips for protecting digital security and privacy in 2023](https://github.com/Lissy93/personal-security-checklist)
* [arkenfox/user.js: Firefox privacy, security and anti-tracking: a comprehensive user.js template for configuration and hardening](https://github.com/arkenfox/user.js)
* [HorlogeSkynet/thunderbird-user.js: Thunderbird privacy, security and anti-fingerprinting: a comprehensive user.js template for configuration and hardening](https://github.com/HorlogeSkynet/thunderbird-user.js)
* [Dig Deeper](https://digdeeper.club/)

**Comment by undergroundwires** [comment]
Other collections shared in GitHub issues:

- https://github.com/undergroundwires/privacy.sexy/issues/258
- https://github.com/undergroundwires/privacy.sexy/issues/203

**Comment by atomGit** [comment]
with regard to Winblows, i would suggest removing support for any proprietary OS for the simple and logical reason that no one can possibly have any reasonable expectation of privacy whilst running proprietary code - sure, individual programs such as the browser can be hardened, but i think this is 99% pointless when the underlying OS cannot be trusted

i know that may sound harsh, but attempting to help those who use proprietary crap can only provide them with a **false sense of privacy/security** and that is potentially quite dangerous

instead, they should be encouraged to dual boot or outright switch to a free OS and provided with any and all help they need to do so

/my $.02

**Comment by ProgrammerBruce** [comment]
Appears to have decent info:
https://www.privacyguides.org
e.g., https://www.privacyguides.org/en/desktop-browsers/
